ReleaseNotes.md

```diff
@@ -1,5 +1,57 @@
 # Release Notes
 
+## New in Release 2.6.0
+
+Post-training Quantization:
+
+- Features:
```

Comment:
I think we've also added FBC support for PyTorch PTQ in this release.

Reply:
Done